tests/test.cpp

```diff
@@ -1681,6 +1681,43 @@ void InvalidFloatTest() {
   TestError("table T { F:float; } root_type T; { F:null }", invalid_msg);
 }
 
+void GenerateTableTextTest() {
+  std::string schemafile;
+  std::string jsonfile;
+  bool ok = flatbuffers::LoadFile((test_data_path + "monster_test.fbs").c_str(),false, &schemafile) &&
+            flatbuffers::LoadFile((test_data_path + "monsterdata_test.json").c_str(), false, &jsonfile);
+  TEST_EQ(ok, true);
+  auto include_test_path = flatbuffers::ConCatPathFileName(test_data_path, "include_test");
+  const char *include_directories[] = { test_data_path.c_str(), include_test_path.c_str(), nullptr };
+  flatbuffers::IDLOptions opt;
+  opt.indent_step = -1;
+  flatbuffers::Parser parser(opt);
+  ok = parser.Parse(schemafile.c_str(), include_directories) &&
+       parser.Parse(jsonfile.c_str(), include_directories);
+  TEST_EQ(ok, true);
+  // Test root table
+  const Monster* monster = GetMonster(parser.builder_.GetBufferPointer());
+  std::string jsongen;
+  auto result = GenerateTextFromTable(parser, monster, "MyGame.Example.Monster", &jsongen);
+  TEST_EQ(result, true);
+  // Test sub table
+  const Vec3* pos = monster->pos();
```

Comment:
The common code style is Google C++ Style. Both declaration are right, second prefferd:
```cpp
Vec3* pos1;
Vec3 *pos2;
```
However, it is better to use the `.clang-format` file for formatting.
What IDE are you using?

Reply:
I am mainly use xCode and Sublime